Configuration/Settings.yaml

```diff
@@ -1,21 +1,36 @@
 MOC:
   ImageOptimizer:
-    useGlobalBinary: FALSE # use globally installed binaries for all formats instead
+    useGlobalBinary: false # use globally installed binaries for all formats instead
     globalBinaryPath: ''
     formats:
-      jpg:
-        enabled: TRUE
-        progressive: TRUE # whether or not to server progressive jpgs
-        useGlobalBinary: FALSE # use globally installed binary for jpegtran instead
-      png:
-        enabled: TRUE
+      'image/jpeg':
+        enabled: true
+        progressive: true # whether or not to serve progressive jpgs
+        useGlobalBinary: false # use globally installed binary for jpeg library instead
+        library: 'jpegtran'
+        binaryPath: 'jpegtran-bin/vendor/jpegtran'
+        arguments: "'-copy none -optimize ' + (progressive ? '-progressive ' : '') + '-outfile ' + file + ' ' + file"
```

Comment:
think it's better to include `${...}` here to make it clear it's Eel, also consistent with https://github.com/neos/neos-development-collection/blob/master/Neos.Neos/Configuration/NodeTypes.yaml#L3